packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -361,14 +361,47 @@ export default abstract class BaseCall implements IWebRTCCall {
     }
   }
 
+  /**
+   * Turns off the audio input track.
+   *
+   * This method will turn of the remote stream audio.
```

Comment:
Maybe it's just me but the wording of the first sentence is confusing. How about using just the second sentence?
```suggestion
   * Turns off the remote stream audio.
```
Also, how does this work with video? Does the video still stream, only the audio is turned off?

Reply:
I didn't test video, I will test and report